docs/_includes/ds6/select.html

```diff
@@ -0,0 +1,70 @@
+<div id="select">
+    <h2><span class="secondary-text">@ebay/skin/</span>select</h2>
+
+    <h3>Default select</h3>
+
+    <div class="demo">
+        <div class="demo__inner" style="max-width: 200px;">
+            <span class="select select__arrow">
```

Comment:
Why do we need to specify `select__arrow` (should be double-dash also, I believe)? Is there a use case for not having the arrow icon?

Reply:
Will move to the `.select` class, and remove the `.select__arrow` class, since the arrow should always be there.